packages/react-instantsearch-hooks-web/src/widgets/HierarchicalMenu.tsx

```diff
@@ -68,6 +72,12 @@ export function HierarchicalMenu({
     canToggleShowMore,
     onToggleShowMore: toggleShowMore,
     isShowingMore,
+    translations: {
+      showMoreButtonText(options) {
```

Comment:
Nitpick (so feel free to disregard), but `isShowingMore` could be destructured.

Reply:
I did not destructure it cause it would shadow the one in the scope above and our ESLint config treats it as an error.
I could destructure and rename but it would be even more verbose

Reply:
Ah, didn't notice the variable already existed.

Current code is fine then 🙂